js/objects/tests/test-id-utils.js

```diff
@@ -15,11 +15,11 @@ describe('Id Utilities tests', () => {
         before(() => {
         });
 
```

Comment:
Maybe we don't need to keep this file at all. I think these tests were originally for the idMaker incrementer etc.